### PR TITLE
[docs] MusicLDM

### LINKED ISSUE #5835: Using pretrained MusicLDM pipeline
### Describe the bug

Hello, in diffusers docs, MusicLDM is used as follow:
```python
from diffusers import MusicLDMPipeline
import torch
import scipy

repo_id = "cvssp/audioldm-s-full-v2"
pipe = MusicLDMPipeline.from_pretrained(repo_id, torch_dtype=torch.float16)
pipe = pipe.to("cuda")

prompt = "Techno music with a strong, upbeat tempo and high melodic riffs"
audio = pipe(prompt, num_inference_steps=10, audio_length_in_s=5.0).audios[0]

# save the audio sample as a .wav file
scipy.io.wavfile.write("techno.wav", rate=16000, data=audio)
```

But the pretrained path  `cvssp/audioldm-s-full-v2` is seem only compatible with AudioLDM, and can not be used with MusicLDM.
Can you fix the usage of MusicLDM

### Reproduction

```python
from diffusers import MusicLDMPipeline
import torch
import scipy

repo_id = "cvssp/audioldm-s-full-v2"
pipe = MusicLDMPipeline.from_pretrained(repo_id, torch_dtype=torch.float16)
pipe = pipe.to("cuda")

prompt = "Techno music with a strong, upbeat tempo and high melodic riffs"
audio = pipe(prompt, num_inference_steps=10, audio_length_in_s=5.0).audios[0]

# save the audio sample as a .wav file
scipy.io.wavfile.write("techno.wav", rate=16000, data=audio)
```

### Logs

_No response_

### System Info

diffusers==0.23.0

### Who can help?

_No response_

### PR DESCRIPTION
**Pull request by stevhliu**
Fixes #5835 to use a compatible checkpoint

### CODE REVIEW

File: src/diffusers/pipelines/musicldm/pipeline_musicldm.py [hunk 1]
```diff
@@ -51,7 +51,7 @@
         >>> import torch
         >>> import scipy
 
-        >>> repo_id = "cvssp/audioldm-s-full-v2"
+        >>> repo_id = "sanchit-gandhi/musicldm-full"
```
Comment (patrickvonplaten) [inline src/diffusers/pipelines/musicldm/pipeline_musicldm.py:54]: Hmm why does the full-v2 not work? We should try to always use the official checkpoints if possible

Comment (sanchit-gandhi) [inline src/diffusers/pipelines/musicldm/pipeline_musicldm.py:54]: This is the 'official' checkpoint: https://huggingface.co/ucsd-reach/musicldm
```suggestion
        >>> repo_id = "ucsd-reach/musicldm"
```

Comment (stevhliu) [reply src/diffusers/pipelines/musicldm/pipeline_musicldm.py]: Thanks, updated with the official checkpoint! :)

**Review by patrickvonplaten — APPROVED** [review]
Thanks!

**Pull request merged**